ala-name-matching-builder/src/main/java/au/org/ala/names/index/TaxonConceptInstance.java

```diff
@@ -979,6 +979,8 @@ private boolean validateParent(Taxonomy taxonomy) {
      */
     private TaxonConceptInstance getResolvedParent(TaxonConceptInstance original, int steps, @Nullable List<TaxonomicElement> trace, boolean exception) {
         if (steps <= 0) {
+            if (original.isForbidden() || this.isForbidden())
```

Comment:
style comment - adding `{}` even if just one statement

Reply:
Yuk. OK fix incoming as soon as it passes its test cases.